Root install guide

Nuclear-Squid opened the pull request:
I wrote a """small""" install root/Arch install guide. I tried to be concise, but I still added about 50 lines (including empty lines) to the file, so if it’s a bit long for you I can try and make it a bit more compact.

I also added the missing `variant` field in some of the layouts, using the same naming convention you used in `prog.toml`

fabi1cazenave commented:
Thanks for the guide ! Would you mind moving it to the `XKalamine` section though ?

It might be unclear, but I try to keep the README in this structure :
- install (`pip install kalamine`)
- layout generation (`kalamine *.toml`)
- layout installation (instructions for every platform)
- XKalamine <= *that* specific tool for which kalamine has to be installed as root

At the end of the README there’s this section, which (I think) would be a good introduction for a `bash` codeblock:

> Using xkalamine with sudo currently supposes kalamine has been installed as root (hopefully in a pyenv). Which really sucks, and we’re working on a better solution.

I prefer putting such information at the end of the README because as you pointed out, there shouldn’t be any need to sudo-run xkalamine for 99% of users. WDYT ?

Nuclear-Squid commented:
Fair enough, I’ll push another commit to fix that

Nuclear-Squid commented:
There, I moved the root-install guide under `XKalamine` and replaced most of the explanations with comments, since it does the job fine and is a lot more compact and easy to read.

I also removed the section that explains how to switch to a layout installed with `xkalamine install` since the command already tells what you should run. I’m all for comprehensive documentations, but I’m sure people will read the message 🙃

fabi1cazenave reviewed (approved):
Thanks a lot !
